ipatests/test_integration/test_trust.py

```diff
@@ -268,6 +253,65 @@ def test_override_homedir(self):
             tasks.restore_files(self.master)
             tasks.clear_sssd_cache(self.master)
 
+    def test_extdom_plugin(self):
+        """Extdom plugin should not return error (32)/'No such object'
+
+        Regression test for https://pagure.io/freeipa/issue/8044
+
+        If there is a timeout during a request to SSSD the extdom plugin
+        should not return error 'No such object' and the existing user should
+        not be added to negative cache on the client.
+        """
+        client = self.clients[0]
+        tasks.backup_file(self.master, paths.SSSD_CONF)
+        log_file = '{0}/sssd_{1}.log' .format(paths.VAR_LOG_SSSD_DIR,
+                                              self.master.domain.name)
+        logsize = len(self.master.get_file_contents(log_file))
+        res = self.master.run_command(['pidof', 'sssd_be'])
+        pid = res.stdout_text.strip()
+        try:
+            tasks.kinit_admin(self.master)
+            # set the extdom plugin timeout to 1s (1000) by default it is 10s
+            entry_ldif = textwrap.dedent("""
```

Comment:
The code will look cleaner if you use master.ldap_connect and methods get_entry/update_entry to read and modify data. You can find example in method disable_autounmembership_in_ldap() in  test_automember.py